ubcpi/ubcpi.py

```diff
@@ -510,9 +510,18 @@ def record_response(self, answer, rationale, status):
         answers = self.get_answers_for_student()
         stats = self.get_current_stats()
         truncated_rationle, was_truncated = truncate_rationale(rationale)
+        corr_ans_text = ''
+        if self.correct_answer == 3:    # handle scenario with no correct answer
```

Comment:
This indicates that the option at the third index was selected by the instructor, for the selection of the correct answer. Index 3 in the options corresponds to "n/a" or no correct answer.